core/provisioning-java/src/main/java/org/apache/syncope/core/provisioning/java/data/UserDataBinderImpl.java

```diff
@@ -668,4 +679,13 @@ public UserTO getUserTO(final String key) {
         return getUserTO(userDAO.authFind(key), true);
     }
 
+    private boolean isMustChangePwdMappingItemConfigured(Mapping mapping) {
```

Comment:
Use `IterableUtils#matchesAny` as in other methods of the same class - and in the whole code, at least in the `2_0_X` branch.